core/commands/object/object.go

```diff
@@ -47,15 +47,6 @@ var ObjectCmd = &cmds.Command{
 		ShortDescription: `
 'ipfs object' is a plumbing command used to manipulate DAG objects
 directly.`,
-		Synopsis: `
-ipfs object data <key>      - Outputs raw bytes in an object
-ipfs object get <key>       - Get the DAG node named by <key>
-ipfs object links <key>     - Outputs links pointed to by object
-ipfs object new <template>  - Create new ipfs objects
-ipfs object patch <args>    - Create new object from old ones
-ipfs object put <data>      - Stores input, outputs its key
-ipfs object stat <key>      - Outputs statistics of object
-`,
```

Comment:
Short help now outputs all subcommands. This exact text is still outputted, but now it happens automatically instead of a human needing to write this by hand in each `command.go` file.

(This falls under "removes redundant cases where subcommands are manually listed in SYNOPSIS (in favour of using autogen everywhere)" from the list up top.)